microprofile-telemetry-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/MPTelemetryServerSetupTask.java

```diff
@@ -0,0 +1,46 @@
+package org.jboss.eap.qe.microprofile.metrics;
+
+import org.jboss.as.arquillian.api.ServerSetupTask;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.eap.qe.microprofile.tooling.server.MicroProfileTelemetryServerSetup;
+import org.jboss.eap.qe.observability.containers.OpenTelemetryCollectorContainer;
+import org.jboss.eap.qe.ts.common.docker.Docker;
+
+/**
+ * Server setup task for configuration of MicroProfile Telemetry and otel collector
+ */
+public class MPTelemetryServerSetupTask implements ServerSetupTask {
+
+    private static OpenTelemetryCollectorContainer otelCollector;
+
+    /**
+     * Start otel collector in container and configure OpenTelemetry and MP Telemetry in application server
+     */
+    @Override
+    public void setup(ManagementClient managementClient, String containerId) throws Exception {
+        // we need a Docker container for The OTel collector here, so throw an exception if a docker service is not available
+        try {
+            Docker.checkDockerPresent();
+        } catch (Exception e) {
+            throw new IllegalStateException("Cannot verify Docker availability: " + e.getMessage());
+        }
+        // start the OTel collector container
+        otelCollector = OpenTelemetryCollectorContainer.getInstance();
```

Comment:
`OpenTelemetryCollectorContainer` lifecycle might be changed by #316, hence this might need to be updated later.